vnext/local-cli/runWindows/utils/deploy.js

```diff
@@ -45,16 +45,25 @@ function getAppPackage(options) {
     options.arch === 'x86'
       ? `{*_x86_${configuration}_*,*_Win32_${configuration}_*}`
       : `*_${options.arch}_${configuration}_*`;
-  const appPackageGlob = `windows/{*/AppPackages,AppPackages/*}/${packageFolder}`;
+
+  const appPackageGlob = path.join(
+    options.root,
+    `windows/{*/AppPackages,AppPackages/*}/${packageFolder}`,
```

Comment:
Using `path.join` on a glob instead of a path feels a bit fragile, since it's not a valid expected input. Could you just concat this yourself like what's already happening?